site/src/content/docs/es/reference/conformance.md

```
@@ -209,13 +209,17 @@ Only **Butterworth** (the library default) and **Chebyshev-II** are class-compli
 </details>
 
 <details>
-<summary>&#9989; <b>Intensity &amp; sound power</b>: 100% (6/6)</summary>
+<summary>&#9989; <b>Intensity &amp; sound power</b>: 100% (10/10)</summary>
 
 | Standard | Quantity | Expected (norm) | Computed | &#916; | Status |
 |:---|:---|:---|:---|:---|:---:|
 | IEC 61043:1994 Clause 5 | Plane-wave intensity I = p^2 / (rho c) | 0.00238 W/m^2 (+/-1.5%) | 0.00239 W/m^2 | 0 W/m^2 | &#9989; |
 | ISO 3744:2010 Eq. 18 | Monopole hemisphere recovers LW (r=4 m) | 95 dB (+/-0 dB) | 95 dB | 0 dB | &#9989; |
 | ISO 9614-2:1996 Eq. 12 | Intensity scan recovers LW of an enclosed source | 90 dB (+/-0.000001 dB) | 90 dB | 0 dB | &#9989; |
+| IEC 61043:1994 Table 2 | Minimum delta_pI0 per band, probe/processor/instrument, class 1/2 | 132 tabulated minima reproduced | max |deviation| 0.000 dB | 0 dB | &#9989; |
```

markdownlint-cli2 (0.23.1)

[warning] Table column count
Expected: 6; Actual: 8; Too many cells, extra data will be missing

(MD056, table-column-count)